.github/workflows/sonarqube.yml

```
@@ -12,16 +12,83 @@ jobs:
+          cp ./backend/coverage.xml ./coverage.xml
+          pwd
+          ls -la
+
       - name: SonarCloud Scan
         uses: SonarSource/sonarcloud-github-action@v2
         with:
-          projectBaseDir: .
-        env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+          projectBaseDir: .
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)